src/components/accessories/patientTherapy/therapyForm/styles.scss

```diff
@@ -1,117 +1,74 @@
 @import "../../../../styles/variables";
 @import "../../../../../node_modules/susy/sass/susy";
-
+$ipadpro: 1024px 1366px;
```

Comment:
This isn't the right place to initialize the variable. Anyway you shouldn't use this but $narrow instead. It should be cover from 1025px to 1200px if I'm right.

Reply:
I will remove it, because the $narrow variable is already there. But the responsive is still unresolved

Reply:
I already fixed that inside another branch which it's not merged yet. The fault was to the Table under the form and not the form itself.